protocols/tests/test_migration/base_test_migration.py

```diff
@@ -1,8 +1,13 @@
 from unittest import TestCase
+from random import randint
+from protocols.reports_3_0_0 import RareDiseaseExitQuestionnaire as RD_EQ_3
 
 
 class TestCaseMigration(TestCase):
 
+    bases = ["A", "C", "G", "T"]
+    chromosomes = range(1, 23) + ["X"] + ["Y"]
```

Comment:
Mitochondrial chromosomes have been very popular lately... we should include them in the test suite. @Antonior26 can you refresh me, what possible values mitochondrial chromosome has in our data? Is it `M` and `MT`?